recipes/nextflow/meta.yaml

```
@@ -17,11 +17,11 @@ source:
 
 requirements:
   host:
-    - conda-forge::openjdk >=11,<=21
+    - conda-forge::openjdk >=17,<=23
```

If the nextflow version didn't change, wouldn't it still support openjdk 11?

---

You're right, this changed in version 24.11.0-edge, so the current version on bioconda (24.10.5, the latest stable) should still support Java 11.

However, support for Java 23 was added in 24.10 so that should be good to update.

```suggestion
    - conda-forge::openjdk >=11,<=23
```

---

Updated in 49b05661a0 to drop the minimum back down to 11.

I added a code comment as a reminder that it'll have to be pushed up to 17 in the next stable release.